packages/jest-types/src/Config.ts

```diff
@@ -28,6 +28,8 @@ export type HasteConfig = {
   platforms?: Array<string>;
   /** Whether to throw on error on module collision. */
   throwOnModuleCollision?: boolean;
+  /** Whether to search for tests in node_modules. */
```

Comment:
Just to keep it in sync with the comment in docs:

```suggestion
/** Whether to retain all files, allowing e.g. search for tests in `node_modules`. */
```